### PR TITLE
feat(RDS): add rds pg account privileges resource

### PR DESCRIPTION
**Pull request by houpeng80**
**What this PR does / why we need it**:
  add rds pg account privileges resource
**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note
  add rds pg account privileges resource
```

## PR Checklist

* [ ] Tests added/passed.

```
make testacc TEST=./huaweicloud/services/acceptance/rds/ TESTARGS='-run TestAccPgAccountPrivileges_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/rds/ -v -run TestAccPgAccountPrivileges_basic -timeout 360m -parallel 4
=== RUN   TestAccPgAccountPrivileges_basic
=== PAUSE TestAccPgAccountPrivileges_basic
=== CONT  TestAccPgAccountPrivileges_basic
--- PASS: TestAccPgAccountPrivileges_basic (752.73s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/rds       752.779s
```

* [ ] Documentation updated.
* [ ] Schema updated.
* [ ] CheckDeleted.

  - **a. During query operation (Read Context)**
    aa. Resource not found
    删除了
    ab. Related resources (parent resources) not found
    \>>>>>> Paste the screenshot here <<<<<<
    -->
  - **b. During delete/disassociate/unbind operation (Delete Context)**
    ba. Resource not found
    \>>>>>> Paste the screenshot here <<<<<<

### CODE REVIEW

File: docs/resources/rds_pg_account_privileges.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
+---
+subcategory: "Relational Database Service (RDS)"
+layout: "huaweicloud"
+page_title: "HuaweiCloud: huaweicloud_rds_pg_account_privileges"
+description: |-
+  Manages an RDS PostgreSQL account privileges resource within HuaweiCloud.
+---
+
+# huaweicloud_rds_pg_account_roles
```
Comment (saf3dfsa) [inline docs/resources/rds_pg_account_privileges.md]:
```suggestion
#huaweicloud_rds_pg_account_privileges
```

File: docs/resources/rds_pg_account_privileges.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+* `user_name` - (Required, String, ForceNew) Specifies the username of the account.
+
+  Changing this will create a new resource.
+
+* `role_privileges` - (Optional, List) Specifies the list of role privileges. Value options: **CREATEDB**,
+  **CREATEROLE**, **LOGIN**, **REPLICATION**.
+
+* `system_role_privileges` - (Optional, List) Specifies the list of system role privileges. Value options:
+  **pg_monitor**,**pg_signal_backend**,**root**.
```
Comment (saf3dfsa) [inline docs/resources/rds_pg_account_privileges.md]:
```suggestion
  **pg_monitor**, **pg_signal_backend**, **root**.
```

Comment (houpeng80) [reply docs/resources/rds_pg_account_privileges.md]: done

File: docs/resources/rds_pg_account_privileges.md (hunk 1 above)
Comment (houpeng80) [reply docs/resources/rds_pg_account_privileges.md]: done

**Pull request merged**